src/omotes_sdk/workflow_type.py

```diff
@@ -704,6 +691,114 @@ def to_pb_value(value: ParamsDictValues) -> float:
             )
 
 
+@dataclass(eq=True, frozen=True)
+class DurationParameter(WorkflowParameter):
+    """Define a datetime parameter this SDK supports."""
+
+    type_name: str = "duration"
+    """Parameter type name."""
+    default: Union[timedelta, None] = field(default=None, hash=False, compare=False)
```

Comment:
Curious if there is any reason for using `Union[timedelta, None]` instead of `Optional[timedelta]` here?

Reply:
Plainly because I copied without looking.. doh! Thanks for the find!

Reply:
Fixed in https://github.com/Project-OMOTES/omotes-sdk-python/pull/68/commits/b3841b3ac320d9013a3f560404463b73a2b12eef